polaris-service/src/main/java/org/apache/polaris/service/persistence/InMemoryPolarisMetaStoreManagerFactory.java

```diff
@@ -35,6 +37,8 @@
 import org.jetbrains.annotations.NotNull;
 
 @JsonTypeName("in-memory")
+@Named("in-memory")
```

Comment:
Here there's CDI added, but also Jackson used as a CDI-ish mechanism.
In CDI, this is a `@Singleton`, but what about Jackson??